sdk/ai/ai-projects/src/agents/utils.ts

```diff
@@ -40,17 +40,154 @@ export enum connectionToolType {
 }
```

Comment:
@ganeshyb Should I remove this enum while we decide what pattern we want to follow for including (or not including) enums?

Reply:
lets keep it, its use in the convenience api's